libevmasm/GasMeter.h

```diff
@@ -43,32 +43,34 @@ class KnownState;
 
 namespace GasCosts
 {
+	/// NOTE: The GAS_... constants referenced by comments are defined for each EVM version in the Execution Specs:
+	/// https://ethereum.github.io/execution-specs/autoapi/ethereum/<evm version>/vm/gas/index.html
```

Comment:
Cancun is still in a branch: https://github.com/ethereum/execution-specs/blob/forks/cancun/src/ethereum/cancun/vm/gas.py.

Also, the links for earlier versions (e.g. [gas docs for shanghai](https://ethereum.github.io/execution-specs/autoapi/ethereum/shanghai/vm/gas/index.html)) are broken temporarily. There was a PR merged an hour ago and something must have gone wrong. Fortunately it can also be viewed in the repo. E.g. [gas docs for shanghai on `master`](https://github.com/ethereum/execution-specs/blob/master/src/ethereum/shanghai/vm/gas.py).